src/components/DigitalCredentials/CredentialsStepper.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <section id="credentials-stepper">
+    <v-container>
+      <v-row>
+        <v-col
+          cols="12"
+          md="8"
+        >
+          <v-card class="pa-3">
+            <v-card-text>
+              <h1 class="mb-4">
+                Steps to get your credentials:
+              </h1>
+              <v-card class="mb-2">
+                <v-card-text class="d-flex">
```

Comment:
A v-card in a v-card is odd. They are usually used to display a white "card" on an otherwise grey page.

I also noticed that your v-cards have a shadow, which is different from our usual style. Please confirm this what your UI/UX team wants.

Reply:
This is what's indicated in the wireframes. Can we maintain it for now?

Reply:
Yes. Resolving.